lambdas/indexer/app.py

```diff
@@ -151,7 +161,43 @@ def update_health_cache(_event: chalice.app.CloudWatchEvent):
     app.health_controller.update_cache()
 
 
-@app.route('/{catalog}/{action}', methods=['POST'])
+@app.route('/{catalog}/{action}', methods=['POST'], method_spec={
+    'tags': ['Indexing'],
+    'summary': 'Notify the indexer to perform an action on a bundle',
+    'requestBody': {
+        'description': 'Contents of the notification',
+        'required': True,
+        **json_content(schema.object(
+            match=schema.object(
```

Comment:
I believe this conflicts with PR #5059 for #5000.

Reply:
That's correct. I'll avoid requesting concurrent reviews for these PRs to avoid conflicts.